agr_indexer/src/main/java/org/alliancegenome/indexer/config/IndexerConfig.java

```diff
@@ -34,6 +35,7 @@ public enum IndexerConfig {
 	ReleaseInfoIndexer("release", ReleaseInfoIndexer.class, 1, 1, 1, 1, 1),
 	DiseaseSummaryIndexer("diseaseSummary", DiseaseSummaryCurationIndexer.class, 4, 1500, 1500, 4, 1),
 	GeneToGeneOrthologyIndexer("geneToGeneOrthology", GeneToGeneOrthologyIndexer.class, 4, 1500, 1500, 8, 1),
+	GOCurationIndexer("goCuration", GOSearchResultCurationIndexer.class, 4, 1500, 1500, 8, 1),
```

Comment:
To be consistant like the gene this should be called goSearchResult

Reply:
@adamgibs sorry I think thinking more of the text name vs Enum name. As in its text name is still "goCuration"